.github/actions/upgrade-instances/action.yaml

```diff
@@ -27,6 +31,7 @@ runs:
 
     - name: Upgrade instances
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
       run: |
         echo "Upgrading instances: ${{ inputs.instances }}"
```

Comment:
We should cd to the working directory and run the rest of the script relative to that